addons/crm/models/crm_recurring_plan.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class RecurringPlan(models.Model):
+    _name = "crm.recurring.plan"
+    _description = "Plan used to calculate recurring revenues"
```

Comment:
Description are generally short and module-prefixed, so maybe "CRM Recurring revenus plans" ?